substrate/zombienet-sdk/tests/staking_next/mod.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::anyhow;
+use subxt::{dynamic, dynamic::Value, ext::scale_value::Composite, OnlineClient, PolkadotConfig};
+use subxt_signer::sr25519::dev;
+use zombienet_sdk::{NetworkConfig, NetworkConfigBuilder};
+
+mod tests;
+
+/// Sets `ValidatorCount` in staking pallet to 500.
```

Comment:
```suggestion
/// Sets `ValidatorCount` in staking pallet to `validator_count`.
```

Reply:
Also worth adding a comment that this is due to some mysterious bug, otherwise it should not be used/needed.

Has there been any updates on that? do we have any new leads as to why it is needed?

Reply:
I've added the comment. I still haven't investigated why the value from the chainspec doesn't get set to the storage item.